rotkehlchen/accounting/types.py

```diff
@@ -101,3 +104,31 @@ def to_db_tuple(self) -> NamedJsonDBTuple:
             ) from e
 
         return event_type, string_data
+
+
+class MissingAcquisition(NamedTuple):
+    asset: Asset
+    time: Timestamp
+    found_amount: FVal
+    missing_amount: FVal
+
+    def serialize(self) -> Dict[str, Union[str, int]]:
+        return {
+            'asset': str(self.asset),
```

Comment:
I missed it before, but since this is for frontend consumption use the identifier here.
```suggestion
            'asset': self.asset.identifier,
```